iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/AbstractCompoundScalable.java

```diff
@@ -71,9 +71,14 @@ public double minimumValue(Network n, ScalingConvention powerConvention) {
         return value;
     }
 
+    @Override
+    public double scale(Network n, double asked, boolean reconnect) {
+        return scale(n, asked, ScalingConvention.GENERATOR, reconnect);
+    }
+
     @Override
     public double scale(Network n, double asked) {
-        return scale(n, asked, ScalingConvention.GENERATOR);
```

Comment:
I don't get this change: I think you must keep the convention and add a default behavior for reconnection. @phiedw I think that reconnection should not be the default behavior, what is you opinion? Why do you need a reconnection in capacity calculation?